partition/roles/ztp/README.md

```diff
@@ -22,50 +22,15 @@ host leaf01 {
 
 For more information on the `ztp.json` format refer to the [documentation](https://github.com/sonic-net/SONiC/blob/master/doc/ztp/ztp.md).
 
-Note that each switch that uses the `ztp.json` file needs an individual `config_db.json`, that it can download at `http://{{ ztp_listen_address }}:{{ ztp_port }}/<hostname>_config_db.json`.
-For example, if the switch's hostname is `r01leaf02`, there should be a file called `r01leaf02_config_db.json` located in `{{ ztp_host_dir_path }}/config/`.
-The configs can be added to the `ztp_additional_files` variable, e.g.
-
-```yaml
-ztp_additional_files:
-  - name: r01leaf02_config_db.json
-    data: "{{ lookup('file', 'path/to/r01leaf02_config_db.json)' | string }}" # using `string` to keep the formatting
-  - name: r02leaf01_config_db.json
-    data: ...
-```
-
-When a SONiC switch is deployed via `ztp.json` and configured by the `sonic` role afterwards, make sure to leave the `sonic_ports`, `sonic_portchannels` and `sonic_breakouts` variables empty and set `sonic_render_config_db_template` to false.
-Otherwise the `sonic` role will override the `config_db.json` provided by the `ztp.json`.
-The result of this may not be intended and, in the worst case, the switch will reach a broken state from which it only can be restored by a factory reset.
-Of course it is also possible to load only a minimal `config_db.json` via ZTP and allow the `sonic` role to render its template based on the `sonic_ports`, `sonic_portchannels` and `sonic_breakouts` variables.
-Both approaches have their pros and cons.
-
-### Pros and Cons of Loading a Static config_db.json via ZTP
```

Comment:
Why you stop using this approach?

Reply:
As @Gerrit91 once pointed out to me, using an approach where large parts of the configuration are applied only once during initial setup and never touched afterwards can add to a certain fear of 'pressing the button' when something needs to be changed. Therefore I decided to go back to the approach where the config is generated on every run of the sonic role.